pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.props

```diff
@@ -13,6 +13,9 @@
         <PackageRID Condition="'$(ArchGroup)' == 'x86' OR '$(ArchGroup)' == 'x64'">win7-$(ArchGroup)</PackageRID>
         <PackageRID Condition="'$(ArchGroup)' == 'arm'">win8-$(ArchGroup)</PackageRID>
         <PackageRID Condition="'$(ArchGroup)' == 'arm64'">win10-$(ArchGroup)</PackageRID>
+        <PackageRID Condition="('$(ArchGroup)' == 'x86' OR '$(ArchGroup)' == 'x64') And '$(TargetGroup)' == 'uapaot'">win7-$(ArchGroup)-aot</PackageRID>
```

Comment:
nit: I would have probably simplified this by defining a new property and setting it to `-aot` for uapaot and to empty for uap to avoid the longer complex conditions and duplication.